skbio/io/format/stockholm.py

```diff
@@ -151,6 +151,39 @@
            }
        }
 
+.. note:: References labelled with ``RN``/``RM``/``RT``/``RA``/``RL``/``RC``
+   are handled differently than other GF features. When reading a Stockholm
+   file with these features, the reader populates adds a dictionary to a list
+   of reference dictionaries, and populates it with the information on that
+   specific reference. If the reference does not include all the expected
+   information (title, author, etc.), the dictionary will only be populated
+   with the information provided.
+
+   A single reference will be stored as::
```

Comment:
It's not necessary to include an example for single vs. multiple references; I'd drop the single reference example and expand the multi-reference example to include the full set of reference tags here.

We provided single- and multi-tree examples above because the structure changes depending on number of trees, but for references it will always be a list of dictionaries.